src/xprs_model.jl

```diff
@@ -387,4 +387,77 @@ function setlogfile(model::Model, filename::String)
         throw(XpressError(model))
     end
     nothing
-end
\ No newline at end of file
+end
+
+mutable struct IISData
```

Comment:
I think the entire IIS thing should live in xprs_iis.jl, as you mentioned there are many ways to improve the interface.